source/styles/font.cpp

```diff
@@ -156,7 +154,11 @@ font &font::name(const std::string &name)
 
 std::string font::name() const
 {
-    return name_.get();
+    if (name_.is_set())
+    {
+        return name_.get();
+    }
+    return "Calibri";
```

Comment:
"magic" constant. Also need to document that this will return a default value instead of throwing if nothing is set. Probably need to mirror this default for font::size also

Reply:
defined as constants